invenio_app_ils/templates/invenio_app_ils/index.html

```diff
@@ -0,0 +1,20 @@
+{# -*- coding: utf-8 -*- Copyright (C) 2018 CERN. invenio-app-ils is free
```

Comment:
Got rid of `base`, `main` and `backoffice` jinja templates, replaced by this one.